apps/web/app/(app)/stats/LargestEmails.tsx

```diff
@@ -90,6 +101,24 @@ export function LargestEmails(props: { refreshInterval: number }) {
                           </Link>
                         </Button>
                       </TableCell>
+                      <TableCell>
+                        <Button
+                          key={item.id}
+                          disabled={item.id ? deletingStatus[item.id] : true}
```

Comment:
`deletingStatus[item.id!]` is fine. These will always have an id. It's just annoying with the gmail api.